### PR TITLE
Refactor.

### PR DESCRIPTION
**Pull request by fzliu**
Linking frontend with backend.

### CODE REVIEW

File: static/js/main.js [hunk 1]
```diff
@@ -1,160 +1,125 @@
...
+      zoomNum = Math.round(imgScale * 100);
+      $("#zoomN").html(zoomNum + "%");
+      return;
+    } else {
+      zoomNum = Math.round(imgScale * 100);
+      $("#zoomN").html(zoomNum + "%");
+    };
+    imgX = imgX * imgScale / imgScale0 - pos.x * (imgScale / imgScale0 - 1);
+    imgY = imgY * imgScale / imgScale0 - pos.y * (imgScale / imgScale0 - 1);
+  }else {
```
Comment (fzliu) [inline static/js/main.js]: 这里加个空白 -> } else {

File: static/js/set.js [hunk 2]
```diff
@@ -26,20 +33,30 @@ function getTagNameset() {
 getTagNameset();
 function changeTagName() {
   $("#confirmN").on ("click", function() {
-    var tagName = $("#selectI").val();
-    var name  = $("#nameT").val();
-    for (var i = 0; i < configSet.tag_names.length; i++) {
-      if (configSet.tag_names[i] == tagName) {
-        configSet.tag_names[i] = name;
-        console.log(configSet);
+    var tagNames = configSet.tags;
```
Comment (fzliu) [inline static/js/set.js]: ";"应该是个","吧

**Comment by fzliu** [comment]
@jkddw 再看下

**Comment by jkddw** [comment]
:+1:

**Pull request merged**